site/components/src/main/java/uk/nhs/digital/intranet/provider/BloomreachSearchProvider.java

```diff
@@ -84,7 +84,7 @@ public int getBloomreachResultsCount(String queryString, SearchArea searchArea)
 
     private HstQuery getQuery(String queryString, SearchArea searchArea) {
         HstQueryBuilder queryBuilder = HstQueryBuilder.create(
-            RequestContextProvider.get().getSiteContentBaseBean().getParentBean());
```

Comment:
What's the effect of this?

Reply:
limits the search results to the Intranet folder